source/build/ukfast_backups.md

```diff
@@ -26,13 +26,12 @@ In order for UKFast to simplify our backup schedule offering without losing the
 
 ### Backup Agents
 
-When referring to Database Backups by default we will apply the below configurations to the following Backups Agents: 
--	Microsoft Exchange 
--	Microsoft SharePoint 
+When referring to Database Backups by default we will apply the below configurations to the following Backups Agents:  
 -	Microsoft SQL Server
 -	PostgreSQL
 -	Oracle DB 
 -	MySQL Server / MariaDB
+-  MongoDB
```

Comment:
If commvault, worth confirming if this is actually working, as last we checked Commvault had only set up one client as was called 'broken' or something

Reply:
Removed until CV send me feedback.